oauth/v1/legacy.go

```diff
@@ -2,14 +2,13 @@ package v1
 
 import (
 	corev1 "k8s.io/api/core/v1"
-	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 var (
 	legacyGroupVersion            = schema.GroupVersion{Group: "", Version: "v1"}
-	legacySchemeBuilder           = runtime.NewSchemeBuilder(addLegacyKnownTypes, corev1.AddToScheme, extensionsv1beta1.AddToScheme)
```

Comment:
where does the v1 for the API live?

Reply:
/lgtm
looks like that turned into the apps API but I don't think we need it in the builder, hopefully

Reply:
@deads2k might shed more light here based on  https://github.com/openshift/api/pull/67/commits/2f6f4d85d80b3f39d66e7e1674aeff71f8a68b3d#diff-a72fa071a28b4a3a648c8e60bd0b288dadc7ff57d1b64ac1244116709f7b452cR12